makefile.cargo

```diff
@@ -10,7 +10,7 @@ CONFIGURE_FLAGS := \
 
     CONFIGURE_FLAGS += \
         --with-default-fonts=/system/fonts \
-        --with-sysroot=$(ANDROID_TOOLCHAIN)/sysroot \
+        --with-sysroot=$(ANDROID_NDK)/platforms/android-18/arch-arm/ \
```

Comment:
Actually, we might not need this line at all; we set the sysroot in the CFLAG here: https://github.com/servo/servo/commit/829a7a9aa3882f1cae585b9fc5c9a6d25ad856da#diff-4b02cf15724590179b4a5025f1a81a0dR268